hydropandas/observation.py

```diff
@@ -868,6 +872,9 @@ def from_bro(
         only_metadata : bool, optional
             if True only metadata is returned and no time series data. The
             default is False
+        engine : str, optional
+            Select how data from the bro-database is obtained, options are 'hydropandas' or
+            'brodata' The default is 'hydropandas'.
```

Comment:
what about brodata_gm?

Reply:
That only works for an `ObsCollection` not for a single observation